src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,10 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+import './PostList.scss';
+
+export const PostList = ({ posts }) => (
```

Comment:
You should define prop types for the PostList component to improve maintainability and readability. PropTypes will also help catch bugs and issues related to wrong data types being passed as props.